examples/spines/spines.py

```diff
@@ -8,6 +8,10 @@
 - normal Axes, with spines on all four sides;
 - an Axes with spines only on the left and bottom;
 - an Axes using custom bounds to limit the extent of the spine.
+
+Each `.axes.Axes` has a list of `~.Spine` objects, accessible
+as ``ax.spines.left``, ``ax.spines.right``, ``ax.spines.bottom``,
+and ``ax.spines.top``.
```

Comment:
`ax.spines.left` and similar is only part of the spine accessor functionality. I suggest to link https://matplotlib.org/stable/api/spines_api.html#matplotlib.spines.Spines instead:

```suggestion
via the container `ax.spines`. See `.Spines`.
```

Reply:
I didn't do the `See .Spines` as we had just linked Spine in the sentence....